plugin/forward/README.md

```diff
@@ -39,7 +39,8 @@ forward FROM TO...
   limited to 15.
 
 Multiple upstreams are randomized (see `policy`) on first use. When a healthy proxy returns an error
-during the exchange the next upstream in the list is tried.
+during the exchange the next upstream in the list is tried. Metadata parameter "forward/resolving_proxy" 
+contains the ip-address and the port of the replied upstream on success, see the log plugin for detail.
```

Comment:
No : the plugins are independents ... it is better to not mention the log plugin here. (IMO)

Reply:
Ok, fixed.